### PR TITLE
Enable ssl for ceph

### PR DESCRIPTION
**Pull request by nicolasbock**

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -2461,6 +2461,15 @@ function enable_ssl_generic()
             fi
             $p "$a['api']['protocol']" "'https'"
         ;;
+        ceph)
+            if ! iscloudver 7plus ; then
+                return
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: I saw these options in cloud6, too - so why not test it there as well?

Comment (nicolasbock) [inline scripts/qa_crowbarsetup.sh]: Good point, I will do that.

**Comment by Itxaka** [comment]
closing and reopening to trigger the travis build as it seems that it was lost in the ether.

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (nicolasbock) [inline scripts/qa_crowbarsetup.sh]: @bmwiedemann You are right, it's also there for cloud6. Where do I find cloud5 though? I mean, what branch in crowbar-ceph is it in?

Comment (nicolasbock) [inline scripts/qa_crowbarsetup.sh]: @bmwiedemann Found it. It's also in Cloud 5.

**Comment by dirkmueller** [comment]
+1 (triggers a new failure though):

```
22:41:44 [2016-08-10T20:41:42+00:00] INFO: Processing service[radosgw] action enable (ceph::radosgw line 45)
22:41:44 [2016-08-10T20:41:42+00:00] INFO: Processing service[radosgw] action start (ceph::radosgw line 45)
22:41:44 
22:41:44 ================================================================================
22:41:44 Error executing action `start` on resource 'service[radosgw]'
22:41:44 ================================================================================
22:41:44 
22:41:44 Chef::Exceptions::Exec
22:41:44 ----------------------
22:41:44 /bin/systemctl start ceph-radosgw@rgw.d52-54-77-77-77-01 returned 1, expected 0
22:41:44 
```

is there some work on fixing this?

**Comment by nicolasbock** [comment]
@dirkmueller Let me look at the problem.

**Comment by nicolasbock** [comment]
I updated https://github.com/crowbar/crowbar-ceph/pull/60, which hopefully addresses the gating failure here.

**Pull request merged**